coresdk/src/coresdk/input.cpp

```diff
@@ -56,8 +56,9 @@ namespace splashkit_lib
         if( ! _input_callbacks.do_quit )
         {
             _input_callbacks.do_quit              = &quit;
-            _input_callbacks.handle_key_down      = &_handle_key_down_callback;
-            _input_callbacks.handle_key_up        = &_handle_key_up_callback;
+            // TODO: Temporarily do not call due to 70c30d4
```

Comment:
I think this will be using the same function pointer declaration as the front end code. We just need to re-declare the function pointer this code is using so that it uses the enum rather than an int. This could keep the front end one as is (with an int parameter), but allow the back end one to change (back to the enum).